models/pardot/model.sql

```diff
@@ -1,4 +1,4 @@
-create or replace view pardot_analysis.visitoractivity_types_meta as (
+create or replace view {schema}.visitoractivity_types_meta as (
```

Comment:
you can use `{...}`  to interpolate values from the config file. Here, we use {schema} to inject `analyst_collective` into the query